slides/06_Templates_Basics.html

```diff
@@ -510,13 +429,11 @@ <h3>Task 6.7 - Template Ref Variable</h3>
                 </ul>
             </section>
             <section>
-                <h3>Task 6.7 - Possible Solution</h3>
+                <h3>Task 6.6 - Possible Solution</h3>
                 <pre>
                      <code class="hljs html" data-trim contenteditable>
-                         &lt;input type="checkbox"
-                                (change)="btn.disabled = !btn.disabled">
-
-                         &lt;button #btn>Button</button>
+                        <bal-checkbox (balChange)="btn.disabled = !btn.disabled"></bal-checkbox>
```

Comment:
Maybe I am confused, but did we said not assignments in the template?

Reply:
discussed - fine like this